diff -rup redhat-config-network-0.9.21.orig/rp3-1.1.10/utils/gtknetmon.c redhat-config-network-0.9.21/rp3-1.1.10/utils/gtknetmon.c
--- redhat-config-network-0.9.21.orig/rp3-1.1.10/utils/gtknetmon.c	Tue Feb 27 00:09:43 2001
+++ redhat-config-network-0.9.21/rp3-1.1.10/utils/gtknetmon.c	Sat Apr  6 12:37:47 2002
@@ -477,7 +477,7 @@ gtk_netmon_realize        (GtkWidget    
 
   /* Load fonts */
 
-  netmon->small_font = gdk_font_load("-schumacher-clean-medium-r-normal-*-*-80-*-*-c-*-iso8859-1");
+  netmon->small_font = gdk_font_load("-schumacher-clean-medium-r-normal-*-*-80-*-*-c-*-iso646.1991-*");
 
   if (netmon->small_font == NULL)
     {

